From 2a80e5ee27ad64fc531d7dda08236426e10b74db Mon Sep 17 00:00:00 2001
From: Harunobu Kurokawa <harunobu.kurokawa.dn@renesas.com>
Date: Mon, 31 Oct 2016 12:50:43 +0900
Subject: [PATCH 231/909] Revert "PCI: rcar: Remove unused platform data"

commit fd6b8014e138335a09fedb9a5f32db0d830b4bcd from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit d9c64c500aecfc1623a9f21e5a563916d4730832.

 In S2RAM code, pcie driver is extracted from private data of device in
rcar_pcie_suspend(). It needs to add pcie as private data for device.

Reported-by: Hien Dang <hien.dang.eb@renesas.com>
Signed-off-by: Harunobu Kurokawa <harunobu.kurokawa.dn@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/pci/controller/pcie-rcar.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/controller/pcie-rcar.c b/drivers/pci/controller/pcie-rcar.c
index fdb38b2004c1..20b2778f3831 100644
--- a/drivers/pci/controller/pcie-rcar.c
+++ b/drivers/pci/controller/pcie-rcar.c
@@ -1154,6 +1154,7 @@ static int rcar_pcie_probe(struct platform_device *pdev)
 	pcie = pci_host_bridge_priv(bridge);
 
 	pcie->dev = dev;
+	platform_set_drvdata(pdev, pcie);
 
 	err = pci_parse_request_of_pci_ranges(dev, &pcie->resources, NULL);
 	if (err)
-- 
2.17.1

